src/verify.c

```diff
@@ -774,11 +774,21 @@ int verify_main(int argc, char **argv)
 		if ((file_not_fixed_count == 0) && (file_not_replaced_count == 0)) {
 			remove_orphaned_files(official_manifest);
 		}
+		if (cmdline_option_picky) {
+			char *start = mk_full_filename(path_prefix, "/usr");
+			fprintf(stderr, "--picky removing extra files under %s\n", start);
+			ret = walk_tree(official_manifest, start, true);
+			if (ret >= 0) {
+				file_checked_count = ret;
```

Comment:
I think it helps understanding there's files that are just bad on your system that should be something else, + there are files on your system swupd didn't add. If anyone else has strong opinions on this though I'm open to them.